starfish/image/_stack.py

```diff
@@ -196,10 +223,17 @@ def show_stack(
             See `Image.get_slice()` for examples.
         color_map : str (default = 'gray')
             string id of a matplotlib colormap
-        rescale : bool (default = True)
-            if True, rescale the data to exclude high and low-value outliers (see skimage.exposure.rescale_intensity)
         figure_size : Tuple[int, int] (default = (10, 10))
             size of the figure in inches
+        show_spots : Optional[SpotAttributes]
+            [Preliminary functionality] if provided, should be a SpotAttribute table that corresponds
+            to the volume being displayed. This will be paired automatically in the future.
+        rescale : bool (default = True)
+            if True, rescale the data to exclude high and low-value outliers (see skimage.exposure.rescale_intensity).
+        p_min: float
+            clip values below this intensity percentile. If provided, overrides rescale, above. (default = None)
```

Comment:
If rescale defaulted to False, then this could throw a ValueError if both are set.

Reply:
Good idea. Will change.